pkg/testfixtures/storage/mysql.go

```diff
@@ -150,8 +150,6 @@ func (m *mySQLTestContainer) RunMySQLTestContainer(t testing.TB) DatastoreTestCo
 	err = mysql.SetLogger(goose.NopLogger())
 	require.NoError(t, err)
 
-	goose.SetLogger(goose.NopLogger())
```

Comment:
This was added because of the verbosity it adds to the test output. It makes our test output cryptic. Can we leave as is?

Reply:
Nono I removed it because the line above it has exactly the same code 🤣